client/landing/stepper/declarative-flow/registered-flows.ts

```diff
@@ -99,6 +101,14 @@ const availableFlows: Record< string, () => Promise< { default: Flow } > > = {
 		import( /* webpackChunkName: "example-flow" */ '../declarative-flow/example' ),
 };
 
+const aiSiteBuilderFlows: Record< string, () => Promise< { default: Flow } > > = config.isEnabled(
+	'calypso/ai-site-builder-flow'
+)
+	? {
+			[ AI_SITE_BUILDER_FLOW ]: () => import( './ai-site-builder' ),
+	  }
+	: {};
+
```

Comment:
I don't think you should bother with a flag, the flow has its own URL and isn't linked anywhere.

Reply:
Well, I wanna be safer just in case we are doing dumb stuff  in the flow, gonna remove it later